segtypes/common/c.py

```diff
@@ -208,7 +208,7 @@ def create_c_file(self, funcs_text, asm_out_dir, c_path):
                     else:
                         c_lines.append("INCLUDE_ASM(\"{}\", {});".format(asm_out_dir / self.name, func_name))
                 else:
-                    asm_outpath = Path(os.path.join(asm_out_dir, self.dir, self.name, func_name + ".s"))
+                    asm_outpath = Path(os.path.join(asm_out_dir, self.name, func_name + ".s"))
```

Comment:
I worry that removing `self.dir` here will prevent the "directory" directive from being used

Reply:
The point is that `asm_out_dir` already includes `self.dir` in the only place this function is called, namely here: https://github.com/ethteck/splat/blob/a942b20507d1550add8ee38792a495a6460c9ebe/segtypes/common/c.py#L112-L122
so it has to be removed either here or there, and to be consistent with the other branches of this if-else, it should be removed here, right?

Reply:
ahhh I see. yeah, ok. makes sense!